app/assets/javascripts/pages.js

```diff
@@ -63,3 +63,11 @@ $('.stats__graph__gem__meter').each(function() {
   bar_width = $(this).data("bar_width");
   $(this).animate({ width: bar_width + '%' }, 700).removeClass('t-item--hidden');
 });
+
+//gem page
+$('.gem__users__mfa-text').on('click', function() {
```

Comment:
Consider putting this into a `$(document).ready(...)` to ensure it runs after the DOM is loaded and the `.gem__users__mfa-text` elements are present.

Reply:
fixed in d68917a